vhd/packer/install-dependencies.sh

```diff
@@ -409,13 +409,26 @@ df -h
 echo "Using kernel:" >> ${VHD_LOGS_FILEPATH}
 tee -a ${VHD_LOGS_FILEPATH} < /proc/version
 { printf "Installed apt packages:\n"; apt list --installed | grep -v 'Listing...'; } >> ${VHD_LOGS_FILEPATH}
-{
-  echo "Install completed successfully on " $(date)
-  echo "VSTS Build NUMBER: ${BUILD_NUMBER}"
-  echo "VSTS Build ID: ${BUILD_ID}"
-  echo "Commit: ${COMMIT}"
-  echo "Feature flags: ${FEATURE_FLAGS}"
-} >> ${VHD_LOGS_FILEPATH}
+
+if [ "$#" -eq 4 ]; then
```

Comment:
is it acceptable a situation where `$#` is not 4?

Reply:
When we call this script from the AIB template, we will ensure 4 arguments are passed into this parameter.